### ISSUE TITLE
Please backport and release ruby 2.7 fixes

### ISSUE THREAD

**Issue by zenspider** [lostisland/faraday#1080]
@tenderlove's PR #1009 was merged but only to master. Can we get it backported to the `0.1x` branch and released? Ruby 2.7 is mere weeks away and this is biting our CI.

**Comment by technoweenie** [comment]
Thanks for the heads-up, I'll take care of it today.

**Comment by technoweenie** [comment]
Alrighty, should be good to go. I'll leave this open until someone can confirm the warnings are gone. 

:turkey:

**Comment by connorshea** [comment]
I was just testing 2.7 and got a lot of this warning:

```
/Users/connorshea/.rbenv/versions/2.7.0-preview3/lib/ruby/gems/2.7.0/gems/faraday-0.17.1/lib/faraday/options.rb:166: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
```

But it looks like that was fixed by c56732cd0df116cc76ec26966eec5b948d04a31c

That's the only deprecation warning from Faraday that I can see. Any chance you could do another release with it? 🙇

**Comment by kapso** [comment]
Curious whats the release date for this fix? Thanks.

**Comment by iMacTia** [comment]
On it 👍

**Comment by iMacTia** [comment]
Unfortunately, I was right to worry 😞: https://github.com/lostisland/faraday/pull/1087/checks?check_run_id=365035952#step:6:74

**Comment by iMacTia** [comment]
I managed to get the CI running for Ruby 2.0+ 🎉
I'm going to merge that in and release 0.17.2

**Comment by iMacTia** [comment]
@technoweenie released 0.17.2 and 0.17.3 so this is now officially available ❤️
